on-boarding/codereview.dart

```diff
@@ -0,0 +1,187 @@
+import 'dart:io';
+
+class Task {
+  String? title;
+  String? description;
+  DateTime dueDate;
+  bool completed;
+
+  Task(this.title, this.description, this.dueDate, {this.completed = false});
+
+  void markAsCompleted() {
+    completed = true;
+  }
+
+  void markAsPending() {
+    completed = false;
+  }
+
+  @override
+  String toString() {
+    String statusEmoji = completed ? "✅" : "🕒";
+    return '📌 Title: $title\n📝 Description: $description\n⏰ Due Date: $dueDate\n$statusEmoji Status: ${completed ? "Completed" : "Pending"}\n';
+  }
+}
+
+class TaskManager {
+  List<Task> tasks = [];
```

Comment:
Consider making the tasks property private, since there are functions to do crud operations to it